packages/ai/cypress/specs/AINoticeIndicator.cy.tsx

```diff
@@ -1,109 +1,162 @@
-		const verificationText = "Verify results before use.";
-
 		it('Should render the component', () => {
 			cy.get('ui5-ai-notice-indicator').should('exist');
 		});
 
-		it("Should display closed responsive poppover", () => {
+		it("Should display closed responsive popover", () => {
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-responsive-popover]")
 				.should("exist");
-
-			cy.get("[ui5-ai-notice-indicator]")
-				.shadow()
-				.find("[ui5-responsive-popover]")
-				.should("have.prop", "open", false);
 		});
 
-		it("Should display correct properties for Default mode", () => {
+		it("Should display correct elements for Default mode", () => {
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.should('have.attr', 'icon', '')
+				.should('not.have.attr', 'icon');
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
 				.should("exist");
+		});
+
+		it("Should display correct elements for Emphasized mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Emphasized');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
+				.should('have.attr', 'icon', 'ai')
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-label]")
-				.contains(verificationText)
 				.should("exist");
 		});
 
-		it("Should display correct properties for Emphasized mode", () => {
-			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Emphasized');
+		it("Should display correct elements for Shortened mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Shortened');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.should('have.attr', 'icon', 'ai')
-				.should("exist");
+				.should('not.have.attr', 'icon');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-label]")
-				.contains(verificationText)
 				.should("exist");
 		});
 
-		it("Should display correct properties for Shortened mode", () => {
-			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Shortened');
+		it('Should display only the AI icon in IconOnly mode', () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'IconOnly');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
-				.find("[ui5-link]")
-				.should('have.attr', 'icon', '')
+				.find("[ui5-button]")
+				.should('have.attr', 'icon', 'sap-icon://ai')
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
-				.should("exist");
+				.should("not.exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
+				.should("be.empty");
```

Comment:
Same here, for IconOnly no Label should be present in the DOM, even if its empty - this part of the test will need to be removed once you fix the rendering